src/App.js

```diff
@@ -18,6 +19,7 @@ function App() {
         {/* Public Routes */}
```

Comment:
You can remove comments after they're no longer needed

Reply:
I used it for the reference.